core/src/main/resources/static/css/table_Modal_Pagination.css

```diff
@@ -1423,22 +1419,23 @@ td input[type=search] {
     border-radius: 5px;
     width: 175px;
     height: 40px;
+    align-content: center;
 }
 
-.btn-secondary:hover {
+.btn.btn-secondary:hover {
```

Comment:
mb also move this colors to some variables in root

Reply:
* this color is already in :root, so i think it would be better to use it as variable.